src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -133,14 +134,18 @@ def parse(self, body):
             return
 
         # Extract commits
-        commits = [
-            '{rev}:{review_request_id}:{diffset_revision}'.format(**c)
+        envs = [
```

Comment:
This is going to increase the workload for mozreview, but I guess we don't really care. Hopefully when we have shallow clones for mozilla-central the impact will be smaller, and mozreview is going to be retired at some point anyways.